configman/tests/test_def_for_argparse.py

```diff
@@ -0,0 +1,477 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from unittest import TestCase
+from mock import patch
+
+from StringIO import StringIO
+
+try:
+    import argparse
+except ImportError:
+    try:
+        from unittest import SkipTest
+    except ImportError:
+        from nose.plugins.skip import SkipTest
+    raise SkipTest
```

Comment:
Mayhaps do this as `raise SkipText("argparse not installed so we can't test def_for_argparse")`